awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupListItem.jsx

```diff
@@ -0,0 +1,178 @@
+import { InstanceGroup } from '../../../types';
+
+const Badge = styled(PFBadge)`
+  margin-left: 8px;
+`;
+
+const ListGroup = styled.span`
+  margin-left: 12px;
+
+  &:first-of-type {
+    margin-left: 0;
+  }
+`;
+
+const Progress = styled(PFProgress)`
+  margin-left: 4px;
+`;
+
+const DataListAction = styled(_DataListAction)`
+  align-items: center;
+  display: grid;
+  grid-gap: 16px;
+  grid-template-columns: 40px;
+`;
+
+function InstanceGroupListItem({
+  instanceGroup,
+  detailUrl,
+  isSelected,
+  onSelect,
+  i18n,
+}) {
+  const labelId = `check-action-${instanceGroup.id}`;
+
+  const isAvailable = item => {
+    return (
+      (item.policy_instance_minimum || item.policy_instance_percentage) &&
+      item.capacity
+    );
+  };
+
+  const isContainerGroup = item => {
+    return item.is_containerized;
+  };
+
+  return (
+    <DataListItem
+      key={instanceGroup.id}
+      aria-labelledby={labelId}
+      id={`${instanceGroup.id} `}
+    >
+      <DataListItemRow>
+        <DataListCheck
+          id={`select-instance-groups-${instanceGroup.id}`}
+          checked={isSelected}
+          onChange={onSelect}
+          aria-labelledby={labelId}
+        />
+
+        <DataListItemCells
+          dataListCells={[
+            <DataListCell
+              key="name"
+              aria-label={i18n._(t`instance group name`)}
+            >
+              <span id={labelId}>
+                <Link to={`${detailUrl}`}>
+                  <b>{instanceGroup.name}</b>
+                </Link>
+              </span>
+            </DataListCell>,
+
+            <DataListCell
+              key="type"
+              aria-label={i18n._(t`instance group type`)}
+            >
+              <b css="margin-right: 24px">{i18n._(t`Type`)}</b>
+              <span id={labelId}>
+                {isContainerGroup(instanceGroup)
+                  ? i18n._(t`Container group`)
+                  : i18n._(t`Instance group`)}
+              </span>
+            </DataListCell>,
+            <DataListCell
+              key="related-field-counts"
+              aria-label={i18n._(t`instance counts`)}
+              width={2}
+            >
+              <ListGroup>
+                <b>{i18n._(t`Running jobs`)}</b>
+                <Badge isRead>{instanceGroup.jobs_running}</Badge>
+              </ListGroup>
+              <ListGroup>
+                <b>{i18n._(t`Total jobs`)}</b>
+                <Badge isRead>{instanceGroup.jobs_total}</Badge>
+              </ListGroup>
+
+              {!instanceGroup.is_containerized ? (
+                <ListGroup>
+                  <b>{i18n._(t`Instances`)}</b>
+                  <Badge isRead>{instanceGroup.instances}</Badge>
+                </ListGroup>
+              ) : null}
+            </DataListCell>,
+
+            <DataListCell
+              key="capacity"
+              aria-label={i18n._(t`instance group used capacity`)}
+            >
+              {!isContainerGroup(instanceGroup) ? (
```

Comment:
I think we should avoid nested ternary operators if we can. You can call a function here and return the JSX

Reply:
I will update this one as well.